jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -1428,20 +1409,24 @@ private String findServerName()
             }
```

Comment:
Do we have coverage for the code above  here?   I.e. does the `findServerName()` method ever find a `Host` header field? as I'm pretty sure that will have already been copied to the authority in the HttpURI by now.... perhaps if a customizer added a Host header but didn't update the HttpURI?

Reply:
Yep, this block is used if a `Customizer` sets a `Host` header without updating the authority.